### ISSUE TITLE
Manual Lyrics scroll

### ISSUE THREAD

**Issue by Shituation** [jacquesh/foo_openlyrics#239]
Be able to manually scroll the lyrics to manualy sync them and stop bad-timed auto scroll

**Comment by Shituation** [comment]
I want the automatic scroll working even when I can manually scroll. I don't want to choose between one or the other

**Comment by regorxxx** [comment]
i.e. if you set it to Automatic scrolling, it should still be possible to use the mouse to scroll overriding it.

As it is right now, it's either fully automatic or fully manual.

**Comment by jacquesh** [comment]
Fair enough, I understand that this is what LSP3 did. I specifically didn't like that though because (as far as I know) there wasn't a way to reset it to default scroll and I didn't want it accidentally getting slightly offset.

I'll think on it. This miiight come in the form of a separate scroll type ("Automatic with scrolling" or something) to allow people to disable it if they don't want it.

**Comment by Tuuuba** [comment]
I found it useful on LSP3 because it let you edit and apply the offset number without having to open the lyric editor, manually type in a new offset and applying it, and the way to reset accidentally scrolling was to just reset the offset to 0.

**Comment by GCRaistlin** [comment]
Another vote for "superseding" auto scroll with manual scroll.

**Comment by oleg-babintsev** [comment]
+1
It's extremely useful feature if you listening radio stream and switching between various channels (when start time of track is unknown for plugin)
